mage_ai/frontend/oracle/components/Table/index.style.tsx

```diff
@@ -3,28 +3,32 @@ import styled from 'styled-components';
 import light from '@oracle/styles/themes/light';
 import { BORDER_RADIUS } from '@oracle/styles/units/borders';
 import { UNIT } from '@oracle/styles/units/spacing';
+import { ScrollbarStyledCss } from '@oracle/styles/scrollbars';
```

Comment:
Alphabetize imports.

Reply:
Ah, is it based on the imported file name (e.g. `'@oracle/styles/scrollbars'`) vs. the imported item name (`ScrollbarStyledCss`)?

Reply:
Based on the variable name, so `ScrollbarStyledCss`  would come before `UNIT`. Not a huge deal though. And we did look into adding an eslint rule, but i think it had organized the imports slightly different than how we do it so we didn't use it. If you know of a way to do it, please feel free to add!